UserSessionsMeter.c

```diff
@@ -0,0 +1,162 @@
+/*
+htop - UserSessionsMeter.c
+(C) 2024 htop dev team
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "config.h" // IWYU pragma: keep
+
+#include "UserSessionsMeter.h"
+
+#include <dlfcn.h>
+#include <utmpx.h>
```

Comment:
This should be moved to some platform implementation as this header does not necessarily exist on all platforms (e.g. WinNT). Putting the actual query part of the code in the `generic` directory is fine though, so most platforms may pull their implementation from there.

Similar things go for the systemd support in this, which are specific to Linux.

Reply:
Moved to *generic/utmpx.(c|h)*.